libs/clients/signature-collection/src/lib/signature-collection.service.spec.ts

```diff
@@ -33,7 +36,7 @@ const sofnun: MedmaelasofnunExtendedDTO[] = [
       id: 123,
       erMedmaelakosning: true,
       kosningTegund: 'Forsetakosning',
-      kosningTegundNr: CollectionType.Presidential,
+      kosningTegundNr: getNumberFromCollectionType(CollectionType.Presidential),
```

Comment:
This looks whack but I tried using declaratively numbered enums. The graphql layer turns those enums into string enums (where the string is their key).